sandbox/tabs.py

```diff
@@ -0,0 +1,147 @@
+from dataclasses import dataclass
+
+from rich.console import RenderableType
+from rich.padding import Padding
+from rich.rule import Rule
+
+from textual import events
+from textual.app import App
+from textual.widget import Widget
+from textual.widgets.tabs import Tabs, Tab
+
+
+class Hr(Widget):
```

Comment:
Not for this PR, but these should first class widgets at some point. I'm sure we will need them in the future.

Reply:
Definitely!